utils/IrodsConnector.py

```diff
@@ -273,11 +272,9 @@ def _get_resources(self):
                 print('    -=WARNING=-    '*4)
         return self._resources
 
-    resources = property(
-        _get_resources, None, None, 'iRODS resource metadata')
-
-    def _get_session(self):
-        """iRODS session getter method.
+    @property
+    def session(self):
```

Comment:
We also should consider splitting this function into several smaller functions

Reply:
Yeah, it is a pretty big method, but there is much for it to do.  I'm very happy that its functionality is removed from the constructor.  I'll think about how best to deal with it.

Reply:
Commit https://github.com/rijksuniversiteit-groningen/iBridges/pull/40/commits/7ac9a2af1f676c17e3f51c6b8bb4cb4a51aec72a should address this.